kerl

```diff
@@ -380,7 +380,7 @@ get_otp_version()
 
 get_perl_version()
 {
-    perl -v | grep version | sed $SED_OPT -e 's/.*v5\.([0-9]+)\.[0-9].*/\1/'
+    SED_OPT=$(perl -v | grep This\ is | perl -pe 's/.*v5\.([0-9]+)\.[0-9].*/\1/')
```

Comment:
Thanks for the PR, the `SED_OPT` variable is something we definitely don't want to change its value.

We are going to solve this issue in a different way by asserting perl exists before we execute a build command.